lib/malloy/core/external.cmake

```diff
@@ -9,6 +9,28 @@ find_package(
     REQUIRED
 )
 
+if (MALLOY_DEPENDENCY_FMT_DOWNLOAD) 
+    FetchContent_Declare( 
+        fmt 
+        GIT_REPOSITORY https://github.com/fmtlib/fmt
+        GIT_TAG 7.1.3 # Highest supported by spdlog 1.8.3 
```

Comment:
We might take this opportunity to upgrade the `spdlog` dependency from `1.8.3` to `1.9.0`. Does this influence the highest supporter version of `fmt`?

Reply:
Yes, that version supports fmt 8

Reply:
Then lets do that.